package.json

```diff
@@ -13,7 +13,7 @@
         "test": "jest",
         "tdd": "jest --watch",
         "pretest": "npm run test:eslint",
-        "start": "NODE_ENV=development parcel src/index.html --open",
+        "start": "NODE_ENV=development parcel src/index.html --open 'Google Chrome'",
```

Comment:
App-navnet er plattformavhengig, vil feile på Windows og Linux

Reply:
Parcel bruker opn/open, og det ser ut til at det skal fungere cross platform 👍 

https://www.npmjs.com/package/opn
https://github.com/parcel-bundler/parcel/pull/978/files

Reply:
```The app name is platform dependent. Don't hard code it in reusable modules. For example, Chrome is google chrome on macOS, google-chrome on Linux and chrome on Windows```

Reply:
D'oh. Personlig har jeg ikke bruk for `--open`, kan alltids klare å skrive inn adressen manuelt hvis jeg ikke har en tab åpen med appen.

Reply:
Samme her, for min del kan vi gjerne fjerne det. Bruker nemlig Safari som default nettleser. I mellomtiden legger jeg til en egen "start-mac"-kommando for at appen åpnes i Chrome.